docs/DesignDocs/Attestation_API_Proposal.md

```diff
@@ -0,0 +1,534 @@
+Proposal of OE SDK Attestation Public and Plugin API for V0.10 Release
+====
+
+# Background
+
+This document describes the proposed OE SDK attestation public and plugin API
+for V0.10 release, as an evolution of the experimental attestation API
+implemented in the
+[OE SDK V0.9 release](https://github.com/openenclave/openenclave/tree/v0.9.x).
+
+This proposal is based on these efforts:
+- Design document [Attestation V3 Update]:
+[Custom Attestation Data Formats for Open Enclave V3 Updates](https://github.com/openenclave/openenclave/blob/master/docs/DesignDocs/CustomAttestation_V3.md).
+- Discussion in [Issue #2729]: [Outstanding issues on attestation plugin design](https://github.com/openenclave/openenclave/issues/2729).
+- Design document [quote-ex Integration]:
+[Attestation: OE SDK Integration with Intel® SGX SDK quote-ex Library for Generation of Evidence in New Formats](https://github.com/openenclave/openenclave/blob/master/docs/DesignDocs/SGX_QuoteEx_Integration.md).
+- Design document [Design Notes]:
+[Notes on OE SDK Attestation API and Plugin Library Design](https://github.com/shnwc/openenclave/blob/master/docs/DesignDocs/NotesOnAttestationAPI.md).
+  - Note: this document is in the process of being up-streamed as OE SDK [PR #2801](https://github.com/openenclave/openenclave/pull/2801).
+
```

Comment:
Can you add https://tools.ietf.org/html/draft-ietf-rats-architecture to the list of efforts this should be based on?

Reply:
Sure.